lib/js/stanford_publication.behavior.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Behavior Example that works with Webpack.
+ *
+ * @see: https://www.npmjs.com/package/drupal-behaviors-loader
+ *
+ * Webpack wraps everything in enclosures and hides the global variables from
+ * scripts so special handling is needed.
+ */
+
+export default {
+
+  // Attach Drupal Behavior.
+  attach: function (context, settings) {
+    (function ($) {
+      $('.publication-topics__collapsable-menu', context).click(function () {
```

Comment:
Lets add a check in here for the browser size or something like that. there's no reason to have the collapsible stuff on desktop.

Reply:
Ok. Take a look and let me know what you think. It is kind of weird if you resize the window.  It is also different from all the other list pages.

Reply:
I'm happy it's different than the others. the others should match what we are trying to do here. 

There are a couple of things that aren't quite right with how you currently have it. the `<button>` tag is still added even on desktop which produces a clickable item that doesn't do anything. I'd prefer to remove the `<button>` wrapping tag entirely on desktop. You are using the `.wrap` function. There's a similar `.unwrap` as well that could accomplish this. https://api.jquery.com/unwrap/

as far as resizing the window, you can add resizing listeners to accomplish that: https://css-tricks.com/working-with-javascript-media-queries/